src/lumigo_tracer/wrappers/__init__.py

```diff
@@ -1,13 +1,16 @@
 from .http.sync_http_wrappers import wrap_http_calls
 from .pymongo.pymongo_wrapper import wrap_pymongo
+from .redis.redis_wrapper import wrap_redis
 
 
 already_wrapped = False
 
 
-def wrap():
+def wrap(force: bool = False):
     global already_wrapped
     if not already_wrapped:
         wrap_http_calls()
+    if force or not already_wrapped:
```

Comment:
Shouldn't it affect `wrap_http_calls`? If not, maybe it's worth adding a comment explaining why and add a docstring for this parameter